qcodes/instrument_drivers/rigol/DG1062.py

```diff
@@ -344,7 +345,7 @@ class DG1062(VisaInstrument):
     waveforms = DG1062Channel.waveforms
 
     def __init__(self, name: str, address: str,
-                 **kwargs: Dict) ->None:
+                 **kwargs) -> None:
```

Comment:
no need for it as it is implicit
```suggestion
                 **kwargs):
```